components/brave_shields/browser/referrer_whitelist_service_browsertest.cc

```diff
@@ -0,0 +1,160 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/path_service.h"
+#include "base/run_loop.h"
+#include "base/task/post_task.h"
+#include "base/test/thread_test_helper.h"
+#include "brave/browser/brave_browser_process_impl.h"
+#include "brave/common/brave_paths.h"
+#include "brave/components/brave_shields/browser/local_data_files_service.h"
+#include "brave/components/brave_shields/browser/referrer_whitelist_service.h"
+#include "chrome/browser/extensions/extension_browsertest.h"
+
+using extensions::ExtensionBrowserTest;
+
+const std::string kLocalDataFilesComponentTestId(
+    "eclbkhjphkhalklhipiicaldjbnhdfkc");
+
+const std::string kLocalDataFilesComponentTestBase64PublicKey =
```

Comment:
Seems to be an unresolved linter error here, components/brave_shields/browser/referrer_whitelist_service_browsertest.cc:21:  For a static/global string constant, use a C style string instead: "const char kLocalDataFilesComponentTestBase64PublicKey[]".  [runtime/string] [4]